state/immutable-trie/trie.go

```diff
@@ -194,7 +194,11 @@ func (t *Trie) Commit(objs []*state.Object) (state.Snapshot, []byte) {
 	nTrie.storage = t.storage
 
 	// Write all the entries to db
-	batch.Write()
+	// TODO, need to handle error
```

Comment:
@DarianShawn add this error handling to the to-do list, the scope of the change API is too large

Reply:
Got it.
We should separate it to some small PRs.